BrowserKit/Sources/ComponentLibrary/Buttons/SecondaryRoundedButton.swift

```diff
@@ -97,6 +98,12 @@ public class SecondaryRoundedButton: ResizableButton, ThemeApplicable {
         setNeedsUpdateConfiguration()
     }
 
+    func addCornerRadiusForVisualEffectView(radiusSize: CGFloat) {
```

Comment:
nit: @dicarobinho add a note here on why this is done for all subviews as in general it is strange to have corner radius for all

Reply:
I am adding the corner radius only for the UIVisualEffectView, it is only one subview. :)

Reply:
I guess my issue is what broke this in first place, why do you have to loop through to find UIVisualEffectView. And in theory you can have more UIVisualEffectView (not that you would in this case) but in theory you can. This is because your where clause finds it all. Lastly curious to know why you don't need a typecast if you are only changing radius of visual effect view (something like below) 
```
for subview in self.subviews where subview is UIVisualEffectView {
    (subview as! UIVisualEffectView).layer.cornerRadius = radiusSize
    subview.clipsToBounds = true 
}
```